eldritch/events.py

```diff
@@ -2718,8 +2722,10 @@ def log(self, state):
     return ""
 
 
-def PassFail(character, condition, pass_result, fail_result):
-  outcome = Conditional(character, condition, "successes", {0: fail_result, 1: pass_result})
+def PassFail(character, condition, pass_result: Event, fail_result: Event, min_successes=1):
```

Comment:
@fingoldfish Thoughts on putting the check difficulty here versus in the check itself somehow? Could always have the PassFail function as something like 
```python
outcome = Conditional(character, condition, "successes", {0: fail_result, condition.difficulty: pass_result})
```

What else should get a difficulty attribute if we go that route?